<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayGraphic2DFrame</class>
 <widget class="QFrame" name="DisplayGraphic2DFrame">
  <property name="enabled">
   <bool>false</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QToolButton" name="toolButtonGeoPlot">
          <property name="text">
           <string>GeoTrace</string>
          </property>
          <property name="icon">
           <iconset resource="../../GPSBook/icons/silk/silk_icons.qrc">
            <normaloff>:/icons/silk/chart_curve.png</normaloff>:/icons/silk/chart_curve.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>16</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
          <property name="arrowType">
           <enum>Qt::NoArrow</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../GPSBook/icons/silk/silk_icons.qrc">
            <normaloff>:/icons/silk/comments.png</normaloff>:/icons/silk/comments.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>16</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonFullfil">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../GPSBook/icons/silk/silk_icons.qrc">
            <normaloff>:/icons/silk/chart_bar.png</normaloff>:/icons/silk/chart_bar.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>16</height>
           </size>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonX">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../GPSBook/icons/silk/silk_icons.qrc">
            <normaloff>:/icons/silk/control_stop.png</normaloff>:/icons/silk/control_stop.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>16</height>
           </size>
          </property>
          <property name="popupMode">
           <enum>QToolButton::InstantPopup</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonY">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../GPSBook/icons/silk/silk_icons.qrc">
            <normaloff>:/icons/silk/style.png</normaloff>:/icons/silk/style.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>16</height>
           </size>
          </property>
          <property name="popupMode">
           <enum>QToolButton::InstantPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonIconOnly</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="toolButtonEdit">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../../GPSBook/icons/silk/silk_icons.qrc">
            <normaloff>:/icons/silk/note_edit.png</normaloff>:/icons/silk/note_edit.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>24</width>
            <height>16</height>
           </size>
          </property>
          <property name="popupMode">
           <enum>QToolButton::InstantPopup</enum>
          </property>
          <property name="autoRaise">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QFrame" name="framePlot">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3"/>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QFrame" name="frameGeoPlot">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </widget>
   </item>
  </layout>
  <action name="actionXAbsoluteTime">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../GPSBook/icons/silk/silk_icons.qrc">
     <normaloff>:/icons/silk/clock.png</normaloff>:/icons/silk/clock.png</iconset>
   </property>
   <property name="text">
    <string>Absolute time</string>
   </property>
  </action>
  <action name="actionXDistance">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../GPSBook/icons/silk/silk_icons.qrc">
     <normaloff>:/icons/silk/flag_blue.png</normaloff>:/icons/silk/flag_blue.png</iconset>
   </property>
   <property name="text">
    <string>Distance</string>
   </property>
  </action>
  <action name="actionXDuration">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../GPSBook/icons/silk/silk_icons.qrc">
     <normaloff>:/icons/silk/clock_add.png</normaloff>:/icons/silk/clock_add.png</iconset>
   </property>
   <property name="text">
    <string>Duration</string>
   </property>
   <property name="toolTip">
    <string>Duration</string>
   </property>
  </action>
  <action name="actionXLongitude">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../GPSBook/icons/silk/silk_icons.qrc">
     <normaloff>:/icons/silk/arrow_right.png</normaloff>:/icons/silk/arrow_right.png</iconset>
   </property>
   <property name="text">
    <string>Longitude</string>
   </property>
   <property name="toolTip">
    <string>Longitude</string>
   </property>
  </action>
  <action name="actionXPoints">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../GPSBook/icons/silk/silk_icons.qrc">
     <normaloff>:/icons/silk/control_stop.png</normaloff>:/icons/silk/control_stop.png</iconset>
   </property>
   <property name="text">
    <string>Points</string>
   </property>
   <property name="toolTip">
    <string>Points</string>
   </property>
  </action>
  <action name="actionYAcceleration">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../GPSBook/icons/silk/silk_icons.qrc">
     <normaloff>:/icons/silk/award_star_bronze_3.png</normaloff>:/icons/silk/award_star_bronze_3.png</iconset>
   </property>
   <property name="text">
    <string>Acceleration</string>
   </property>
  </action>
  <action name="actionYAltitude">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../GPSBook/icons/silk/silk_icons.qrc">
     <normaloff>:/icons/silk/style.png</normaloff>:/icons/silk/style.png</iconset>
   </property>
   <property name="text">
    <string>Altitude</string>
   </property>
  </action>
  <action name="actionYDistance">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../GPSBook/icons/silk/silk_icons.qrc">
     <normaloff>:/icons/silk/flag_blue.png</normaloff>:/icons/silk/flag_blue.png</iconset>
   </property>
   <property name="text">
    <string>Distance</string>
   </property>
  </action>
  <action name="actionYSpeed">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../GPSBook/icons/silk/silk_icons.qrc">
     <normaloff>:/icons/silk/car.png</normaloff>:/icons/silk/car.png</iconset>
   </property>
   <property name="text">
    <string>Speed</string>
   </property>
  </action>
  <action name="actionYLatitude">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../GPSBook/icons/silk/silk_icons.qrc">
     <normaloff>:/icons/silk/arrow_up.png</normaloff>:/icons/silk/arrow_up.png</iconset>
   </property>
   <property name="text">
    <string>Latitude</string>
   </property>
  </action>
  <action name="actionSplitSegment">
   <property name="icon">
    <iconset resource="../../GPSBook/icons/gtk/gtk_icons.qrc">
     <normaloff>:/icons/gtk/edit-cut.png</normaloff>:/icons/gtk/edit-cut.png</iconset>
   </property>
   <property name="text">
    <string>Split segment</string>
   </property>
   <property name="toolTip">
    <string>Split segment</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionDelete_selected_point">
   <property name="icon">
    <iconset resource="../../GPSBook/icons/gtk/gtk_icons.qrc">
     <normaloff>:/icons/gtk/object-cancel.png</normaloff>:/icons/gtk/object-cancel.png</iconset>
   </property>
   <property name="text">
    <string>Delete selected point</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionXCumulDistance">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../GPSBook/icons/silk/silk_icons.qrc">
     <normaloff>:/icons/silk/flag_red.png</normaloff>:/icons/silk/flag_red.png</iconset>
   </property>
   <property name="text">
    <string>Cumul Distance</string>
   </property>
  </action>
  <action name="actionYCumulDistance">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../../GPSBook/icons/silk/silk_icons.qrc">
     <normaloff>:/icons/silk/flag_red.png</normaloff>:/icons/silk/flag_red.png</iconset>
   </property>
   <property name="text">
    <string>Cumul Distance</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../GPSBook/icons/gtk/gtk_icons.qrc"/>
  <include location="../../GPSBook/icons/silk/silk_icons.qrc"/>
 </resources>
 <connections/>
</ui>
